src/settings.ts

```diff
@@ -79,3 +79,4 @@ export enum METRIC_NAMES {
   REQUEST_INVALID = 'request_invalid',
   REQUEST_NOT_CREATED = 'request_not_created',
 }
+
```

Comment:
this was just to trigger the tests to rerun when it had a weird codecov error and wouldn't rerun